### PR TITLE
Built items controller

### PR DESCRIPTION
**Pull request by Aram-Anderson**
# Issue Number: 2

# Description of changes: Created an items controller

# @mentions of the person or team responsible for reviewing proposed changes:
@dphilla @mikeyduece @liambarstad

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class ItemsController < ApplicationController
+
+  def index
+    @items = Item.all
```
Comment (mikeheft) [inline app/controllers/items_controller.rb:4]: looks good.

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  # config.filter_gems_from_backtrace("gem name")
+  #
+
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+
+  config.use_transactional_fixtures = true
+
+  config.infer_spec_type_from_file_location!
+
+  config.filter_rails_from_backtrace
```
Comment (mikeheft) [inline spec/rails_helper.rb:71]: Need to add `!` here

**Comment by mikeheft** [comment]
@Aram-Anderson looks good. I left a comment that needs to happen or tests will fail on line 68 of the `rails_helper`

File: app/controllers/items_controller.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  end
```
Comment (liambarstad) [inline app/controllers/items_controller.rb:5]: do we have a view for this yet?

**Pull request closed without merging**